app/helpers/admin/users_helper.rb

```diff
@@ -89,5 +89,25 @@ def organization_tooltip(organization_names, count, imposed_limit: 0)
         str + " & #{overflow_count(count, imposed_limit: imposed_limit)} others"
       end
     end
+
+    # Returns a boolean that indicates whether a user can be banished
+    # A user can be banished when:
+    # - the users account was created in the period before the last 100 days ago
+    # and there are no comments from the user in the period before the last 100 days.
+    # - if the logged in user is a super admin or support admin
+
+    # Some practical scenarios:
+    # Today is the 06 July:
+    # - if a user was created before the 27 March 2022 (101 days ago)
+    # and they had no comments before the 27 March 2022 then they can be banished.
+    # - if they have any comments before the 27 March 2022 then they cannot be banished
+    # - if they were created before the 27 March 2022 (101 days ago) and they have comments
+    # before the 27 March 2022 then they cannot be banished.
+    #  @param {Array} [array] the user
+    # @return [Boolean]
+    def banishable_user?(user)
+      (user.comments.where("created_at < ?", 100.days.ago).empty? && user.created_at < 100.days.ago) ||
+        current_user.super_admin? || current_user.support_admin?
```

Comment:
Reading Note: We are allowing the banishment of `super_admins` and `support_admins`?  This logic carries forward from prior implementations but might be a bit surprising.

Reply:
I read this as: `a_user` is banishable if `current_user` is a `super_admin` or `support_admin`.

I sort of wonder if this might belong in a `policy`? Other than `current_user`, it feels model-ish (and policies generally tend to feel model-ish and have access to `current_user`).

Reply:
> I sort of wonder if this might belong in a policy?

Possibly - my first instinct was to drop this into the model but then realized that the check includes the `current_user`. I'll look into this. Thanks!